app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -64,6 +64,19 @@ def connect(options = {})
     self.class.raw_connect(server, port, username, password, service)
   end
 
+  def metrics_connect(options = {})
```

Comment:
@chessbyte I don't thing so because we don't call verify_credentials_for_rhevm when we connect to the service or inventory.
@Fryguy or @blomquisg Thoughts?